app/assets/src/components/visualizations/GenomeViz.js

```diff
@@ -118,14 +118,6 @@ export default class GenomeViz {
       }
     }
 
-    if (this.options.onGenomeVizHover) {
```

Comment:
why did this go away?

Reply:
It was replaced with `this.options.onGenomeVizBarHover`, but forgot to remove.